cmd/api/src/api/saml/saml.go

```diff
@@ -266,29 +269,71 @@ func domainValue(host url.URL) string {
 	}
 }
 
+func (s ProviderResource) auditSAMLLogin(requestContext context.Context, user model.User, status model.AuditLogEntryStatus, err error) error {
```

Comment:
It looks like we already have [auditLogin](https://github.com/SpecterOps/BloodHound/blob/96380ddb214a4034d1dbf26bbeb15223fd390fbf/cmd/api/src/api/auth.go#L92) that we could perhaps export and wrap since the functionality seems the same.